kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/FilterDefinition.java

```diff
@@ -15,7 +15,7 @@
 import io.kroxylicious.proxy.plugin.PluginImplName;
 
 public record FilterDefinition(
-                               @PluginImplName(FilterFactory.class) @JsonProperty(required = true) String type,
+                               @PluginImplName(FilterFactory.class) @JsonProperty(required = true, defaultValue = "") String type,
```

Comment:
what does required + defaultValue mean?

Reply:
That I tried something and forgot to revert it, thanks for catching.